### DISCUSSION TITLE
【羊村】奖励机制【笔记+翻译+校验+宣传+书评】

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/home#243] [Announcements]
## 更新时间：2024-05-17

> 文本奖励：

- 文章翻译：
    - 超短1元/章，正常文章2～3元不等（代码不算在内）（可以采用Google翻译+自行校验的方式）
    - 外加奖励（每10篇再奖励8元 / 6篇奖励4元)
- 文章校验：
    - 文章按照修正量来计算，正常可以给 1元/篇（修改量较大的，按照翻译的奖励算！）
    - 外加奖励（每10篇再奖励4元 / 6篇奖励2元； 如果修改了较大，可以翻倍奖励）
- 目前任务有： 
    - [pytorch](https://pytorch.apachecn.org): <https://github.com/apachecn/pytorch-doc-zh>
        -  pytorch 奖励系数变为 1.5，例如 2元的，变成 2*1.5 = 3元！（翻译 和 校验 都乘 1.5的系数）
        - 更新时间：2024-05-17
    - [huggingface](https://huggingface.apachecn.org): <https://github.com/apachecn/huggingface-doc-zh>
    - 下面待定
    - [LangChain](https://langchain.apachecn.org): <https://github.com/apachecn/langchain-doc-zh>
    - [Kafka](https://kafka.apachecn.org): <https://github.com/apachecn/kafka-doc-zh>
    - [sklearn](https://sklearn.apachecn.org): <https://github.com/apachecn/sklearn-doc-zh>
 - 奖励保证不亏待人！（不要红包，可以换其他的奖励，私聊）

> 原创笔记：

- 金融、IT互联网、财商教育、个人笔记【分好类别，尽量是开源项目，别出现版权问题就行】
- 原创内容：奖励的基础上 x 1.5倍/2倍

> 翻译奖励：

- 按照文本奖励来执行！
- 要求：
    - 可以参考： Google翻译、ChatGPT、已经翻译过的文档 （要求校验 + 至少了解 并且 使用过该框架）

> 媒体宣传：

- 各大媒体平台：3篇文章1元，人均一天限制30篇（单个项目最多10篇）
- 主流媒体都行：博客园/CDSN/开源中国/B站/微博

> 书评奖励

各位，收集每个老哥看的：金融相关的书 + 评分（1～5分） + 评价内容（30字+）
每条内容：1～5元红包不等（价格可以让我或者群友评价，最后选择一个最高价格）。每个人每天限制 10条。

要求：
1. 不需要和这里重复就行： https://docs.qq.com/sheet/DVm1XWHZTSmN5bG9z?tab=BB08J2
2. 一定要是自己看过的心得

例如：
--- : 书名 - 债务危机 
--- : 评分 - n颗星 (1/5)
--- : 读者 - 八岁的兔斯基
--- : 书评 - 理解经济的循环，膨胀和经济发展的关系，后续可以和唐朝的手把手财报18节银行股对应上。如何膨胀如何处理债务，如何在债务期下继续发展。

> 适合人群：

- 喜欢开源
- 乐于分享
- 时间充裕
- 想赚零花钱

---

- 如果大家对奖励有什么困惑，可以找我说，我随时调整（可长期保持合作）
- 推荐朋友参与的奖励：该朋友半年内 1%～5%的收益（最高不超过100元）

**Comment by sanxincao** [comment]
好好好

**Comment by jiangzhonglian** [comment]
1
